infrastructure/src/main/java/org/corfudb/infrastructure/health/Component.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.infrastructure.health;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Component is the infrastructure service for which the health is tracked
+ */
+public enum Component {
```

Comment:
any plan to add Log Replicator to this list?

Reply:
This is just the first iteration. The developers can add the components and issues they want to be reported any time